diff -u -r pkg-clean/kolide-k2-launcher/etc/kolide-k2/launcher.flags pkg/kolide-k2-launcher/etc/kolide-k2/launcher.flags
--- pkg-clean/kolide-k2-launcher/etc/kolide-k2/launcher.flags	2023-08-03 16:19:08.000000000 -0500
+++ pkg/kolide-k2-launcher/etc/kolide-k2/launcher.flags	2023-08-03 16:22:23.205482212 -0500
@@ -4,5 +4,5 @@
 update_channel stable
 transport jsonrpc
 hostname k2device.kolide.com
-root_directory /var/kolide-k2/k2device.kolide.com
-osqueryd_path /usr/local/kolide-k2/bin/osqueryd
+root_directory /var/lib/kolide-k2/k2device.kolide.com
+osqueryd_path /opt/kolide-k2/bin/osqueryd
diff -u -r pkg-clean/kolide-k2-launcher/usr/lib/systemd/system/kolide-k2-launcher.service pkg/kolide-k2-launcher/usr/lib/systemd/system/kolide-k2-launcher.service
--- pkg-clean/kolide-k2-launcher/usr/lib/systemd/system/kolide-k2-launcher.service	2023-08-03 16:19:08.000000000 -0500
+++ pkg/kolide-k2-launcher/usr/lib/systemd/system/kolide-k2-launcher.service	2023-08-03 16:23:35.230282497 -0500
@@ -1,13 +1,12 @@
 [Unit]
-Description=The Kolide Launcher
+Description=Kolide K2 Launcher
 After=network.service syslog.service
 
 [Service]
-ExecStart=/usr/local/kolide-k2/bin/launcher \
--config \
-/etc/kolide-k2/launcher.flags
+ExecStart=/opt/kolide-k2/bin/launcher -config /etc/kolide-k2/launcher.flags
 Restart=on-failure
 RestartSec=3
 
 [Install]
-WantedBy=multi-user.target
\ No newline at end of file
+WantedBy=multi-user.target
+
